source/core/Math.cpp

```diff
@@ -341,7 +341,9 @@ DECLARE_SCALE2X_INTN_HELPER(Single)
     VIREO_FUNCTION_SIGNATURE7(InRangeAndCoerce##TYPE, TYPE, TYPE, TYPE, Boolean, Boolean, TYPE, Boolean) { \
          /* Args:  x loLimit hiLimit includeLo includeHi coercedOut  inRangeOut */\
         VIVM_TRACE_FUNCTION(InRangeAndCoerce##TYPE)    \
-        _Param(5) = _Param(0) < _Param(1) ? _Param(1) : _Param(0) > _Param(2) ? _Param(2) : _Param(0); \
+        if (_ParamPointer(5)) { \
+            _Param(5) = _Param(0) < _Param(1) ? _Param(1) : _Param(0) > _Param(2) ? _Param(2) : _Param(0); \
+        } \
         _Param(6) = (_Param(0) > _Param(1) || (_Param(3) && _Param(0) == _Param(1)))    \
```

Comment:
Only _Param(5) is optional (wildcard) during CodeGen. _Param(6) is always used. It is this primitive specific for the CompareAggregates case.